source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -53,41 +58,41 @@ int main()
     try
     {
         std::cout << std::setprecision(4);
-        Eigen::IOFormat MatrixFmt(4, 0, ", ", "\n", "[", "]", "[", "]");
-        Eigen::IOFormat VectorFmt(4, 0, ", ", "", "", "", "[", "]");
+        Eigen::IOFormat matrixFmt(4, 0, ", ", "\n", "[", "]", "[", "]");
```

Comment:
matrixFormat?

Reply:
Well, it's not an abbreviation just for "matrix format". It's "matrix formatting rules" (or "vector formatting rules")

Reply:
Exactly :) It misslead me to thinking it was something else. So matrixFormattingRules then.

Reply:
ok